data/strings/types_strings.txt

```diff
@@ -24227,6 +24701,7 @@
     fa = هتل
     fi = Hostelli
     fr = Auberge de jeunesse
+    hi = पथचारी आन्दोलन
```

Comment:
```suggestion
    hi = शयनागार|छात्रावास|होस्टल
```

Reply:
are these synonyms? Which one of these should be seen in the app? I'll add the other two to categories.txt where search terms are stored.

Reply:
Yes, kind of synonyms. in this context, the 1st of 3 is best suited to be seen in the app.